src/elvis_style.erl

```diff
@@ -304,9 +305,23 @@ result_node_line_col_fun(Msg) ->
 
 %% Rule checking
 
+-spec line_is_comment(binary()) -> true | false.
+line_is_comment(Line) ->
+    case re:run(Line, "^ *%") of
```

Comment:
Only looking at whole-line comments was, yes, a purposeful decision. It may just be a personal thing for me, but if we wanted to give more power to the rule we could go from accepting a boolean value to something like:

```
skip_comments = false | whole_line | any
```

Then for the `any` value we would just use a regex to remove line-end comments before evaluating length. Let me know if you'd like to see that now or not.